src/Sylius/Bundle/PaymentBundle/Resources/config/workflow/state_machine.yaml

```diff
@@ -0,0 +1,49 @@
+framework:
```

Comment:
Winzou and workflow SMs missing `authorize` transition and `authorized` state. Both of them are defined at interfaces, should they be added to those definition now? Personally I would prefer to have them if they are defined.

Reply:
#15423